### DISCUSSION TITLE
Building for aarch64

### DISCUSSION THREAD

**Discussion by ratkinkrf** [DOCGroup/ACE_TAO#2290] [Q&A] [answered]
I'm trying to make rpm packge and noticed that in spec you dont set buildbits=64 for aarch64 and setting it is pointless as it will set -m64 which will cause error.Do I need to set something else for it or here https://github.com/DOCGroup/ACE_TAO/blob/ae8f47ce8be00e9ecc7d197c2923a4f7ac5c3195/ACE/ace/config-linux-common.h#L181 is that everything needed

**Comment by mitza-oci** [comment, marked as answer]
I don't know of any changes or special configuration needed for AArch64.  Typically the compiler is set up for its target platform and then ACE can "detect" it based on these predefined preprocessor symbols (referencing the line you quoted).
